hs_core/models.py

```diff
@@ -1185,9 +1186,8 @@ def bag_url(cls, resource_id):
     @classmethod
     def scimeta_url(cls, resource_id):
         scimeta_path = "{resource_id}/data/resourcemetadata.xml".format(resource_id=resource_id)
-        istorage = IrodsStorage()
-        scimeta_url = istorage.url(scimeta_path)
-
+        scimeta_url = reverse('rest_download',
+                          kwargs={'path': scimeta_path})
```

Comment:
@hyi You can add the comment `  # noqa` to the end of the line for things that we truly want the linter to ignore.